test/typings/test-d/where.test-d.ts

```diff
@@ -25,6 +25,17 @@ function testWhere(db: Kysely<Database>) {
   // Nullable LHS with reference
   db.selectFrom('person').whereRef('last_name', '=', 'first_name')
 
+  // ColumnType reference and non-ColumnType reference.
```

Comment:
Unrelated. Leftovers from my investigation on dropping `SelectType<T>` from `ref`. Thought these would be useful tests regardless.